website/contributing/release-testing.md

```diff
@@ -74,8 +75,21 @@ Followed by the wanted options:
   -t, --target      [choices: "RNTester", "RNTestProject"] [default: "RNTester"]
   -p, --platform                    [choices: "iOS", "Android"] [default: "iOS"]
   -h, --hermes                                         [boolean] [default: true]
+  -c, --circleciToken                                                   [string]
 ```
 
+Using the `-c <your-token>` option is recommended, as the script will download the required artifacts from CircleCI, instead of having to build from source reducing the testing time
+There are a couple of extra requirements to make this flow work:
+1. the CI must have run at least once in the current branch. We need to download artifacts generated by CircleCI, so we need to have at least one run of the CI.
```

Comment:
Can we add details for where to check? Or does the script provide details as part of the error message?

Reply:
Currently, it tells you which job is it expecting to. I think we can have it output the link to the pipeline if we think that this is helpful.